.alexrc

```diff
@@ -1,3 +1,3 @@
 {
-  "allow": ["basically", "bigger", "corruption", "corruptions", "crash", "dead", "desire", "DIX", "easily", "easy", "execute", "executed", "execution", "failed", "failure", "failures", "garbage", "harder", "host-hostess", "invalid", "just", "lies", "man", "mero", "obvious", "period", "periods", "que", "reject", "remains", "sane", "simple", "special", "traps", "Xyratex"]
+  "allow": ["basically", "bigger", "clovis", "corruption", "corruptions", "crash", "dead", "desire", "dix", "DIX", "easily", "easy", "execute", "executed", "executes", "execution", "executions", "failed", "failure", "failures", "fellow", "gals-man", "garbage", "hack", "harder", "hooks", "host-hostess", "hostesses-hosts", "hosts", "invalid", "just", "lies", "man", "mero", "obvious", "period", "periods", "que", "reject", "remain", "remains", "sane", "simple", "special", "trap", "traps", "Xyratex"]
```

Comment:
@bdekvadiya,  I did not find any word pairs like 'hostesses-hosts' or 'gals-man' when I grepped for it. Where do you see these words being used?

Reply:
@mehjoshi the alex is scanning and reporting the words host and hosts for these words to be addressed we have added "hostesses-hosts" which is the workaround @subhalaxmisa has suggested, also the word "man" is removed from alex report  in one of the file if we add "gals-man" in the exception list.
![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/86288324/175230414-dbf8f568-5635-40b2-be49-3935b625b8ff.png)
![MicrosoftTeams-image (6)](https://user-images.githubusercontent.com/86288324/175230479-e5208f7d-2317-4f93-8cfe-c269855cee3c.png)